Artifacts/windows-azurepowershell/StartChocolatey.ps1

```diff
@@ -43,7 +43,11 @@ $scriptContent = Get-Content -Path $command -Delimiter ([char]0)
 $scriptBlock = [scriptblock]::Create($scriptContent)
 
 # Run Chocolatey as the artifactInstaller user
-Enable-PSRemoting �force
+Enable-PSRemoting –force -SkipNetworkProfileCheck
+
+# Ensure that current process can run scripts. 
+Set-ExecutionPolicy -ExecutionPolicy Bypass -Scope Process -Force 

```

Comment:
Is "-ExecutionPolicy bypass" needed here? Since the invoked script block (Artifacts/windows-azurepowershell/InstallViaWebPICmd.ps1) also seems to set it.